packages/@lwc/template-compiler/src/codegen/codegen.ts

```diff
@@ -207,6 +209,42 @@ export default class CodeGen {
         return t.conditionalExpression(bindExpr, t.literal(''), t.literal(null));
     }
 
+    genSanitizedHtmlExpr(expr: t.Expression) {
+        const instance = this.innerHtmlInstances++;
+
+        this.usedLwcApis.add('sanitizeHtmlContent');
+
```

Comment:
The generated code is a bit counterintuitive here. Might be helpful to add a comment:

```suggestion
// Example input: <div lwc:inner-html={foo}>
// Output: $ctx._rawHtml$0 !== ($ctx._rawHtml$0 = $cmp.foo)
             ? ($ctx._sanitizedHtml$0 = sanitizeHtmlContent($cmp.foo))
             : $ctx._sanitizedHtml$0
```